packages/cli/src/init.ts

```diff
@@ -242,7 +252,11 @@ export function generateScript(project: { name: string }): string {
 export function generateTSConfig(): string {
     return detent(`
         {
-            "extends": "@gitbook/tsconfig/integration.json"
+            "extends": "@gitbook/tsconfig/integration.json",
+            "compilerOptions": {
+            "lib": ["ES6", "DOM"],
+            "types": ["@cloudflare/workers-types"]
```

Comment:
Which `integration.json` are you referring to? I was getting some linting and type errors without having these in the project 😄

Reply:
From `@gitbook/tsconfig`
```
{
    "extends": "@gitbook/tsconfig/integration.json"
}
```

Reply:
Ah yes - I see that now. 

It wasn't being picked up before because the dependency wasn't specified in `devDependencies`. Will remove, thanks for flagging!